infra/base-images/base-builder/test/detect_repo_test.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019 Google LLC
```

Comment:
Why did we need to move this?
Also, Did you delete the old version of this?

Reply:
Ah, because of pycache
https://github.com/google/oss-fuzz/pull/5986#discussion_r690719246

CI would come up with errors like:
```
==================================== ERRORS ====================================
________ ERROR collecting base-images/base-builder/bisect_clang_test.py ________
import file mismatch:
imported module 'bisect_clang_test' has this __file__ attribute:
  /home/runner/work/oss-fuzz/oss-fuzz/infra/base-images/base-builder-new/bisect_clang_test.py
which is not the same as the test file we want to collect:
  /home/runner/work/oss-fuzz/oss-fuzz/infra/base-images/base-builder/bisect_clang_test.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
________ ERROR collecting base-images/base-builder/detect_repo_test.py _________
import file mismatch:
imported module 'detect_repo_test' has this __file__ attribute:
  /home/runner/work/oss-fuzz/oss-fuzz/infra/base-images/base-builder-new/detect_repo_test.py
which is not the same as the test file we want to collect:
  /home/runner/work/oss-fuzz/oss-fuzz/infra/base-images/base-builder/detect_repo_test.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
=========================== short test summary info ============================
```

Reply:
also done with the old version